chart/Chart.yaml

```diff
@@ -16,6 +16,10 @@ dependencies:
     alias: internalRedis
     repository: https://charts.bitnami.com/bitnami
     version: 16.x.x
+  - name: replicated
+    repository: oci://registry.replicated.com/library
+    version: 1.0.0-beta.13
+    condition: replicated.enabled
```

Comment:
What is going to happen if we don't set that value on `values.yaml`?
We should avoid it on regular installacion, right?

Reply:
you are right, we have to disable in the `values.yaml` file